haas/client/auth.py

```diff
@@ -0,0 +1,13 @@
+import base64
+
+
+
+def auth_db(username, password):
```

Comment:
To avoid reinvention, after #615 merges I would suggest copying the various HTTPClient() implementations from cli.py. Eventually, when cli.py is completely calling your client library, we can then remove the original HTTPClient().